packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/TableSelector.tsx

```diff
@@ -15,20 +17,28 @@ import { Popover, PopoverContent, PopoverTrigger } from "../../../../../../compo
 import { cn } from "../../../../../../lib/utils";
 
 type Props = {
-  value: string | undefined;
-  options: string[];
+  tables: string[];
 };
 
-export function TableSelector({ value, options }: Props) {
+export function TableSelector({ tables }: Props) {
+  const [selectedTableId, setTableId] = useQueryState("tableId");
   const [open, setOpen] = useState(false);
   const { worldAddress } = useParams();
 
+  useEffect(() => {
+    if (!selectedTableId && tables.length > 0) {
+      setTableId(tables[0] as Hex);
```

Comment:
does this operation add another history item? if so that might make the back button a little hard to use, where each time you hit back, it triggers this effect again which adds to history, making back a bit of a no-op unless you can skip past the page where there's no table ID set

ideally this operation would use `replace` here

Reply:
The default for `nuqs` is `replace`